lib/_http_server.js

```diff
@@ -802,7 +802,7 @@ function resOnFinish(req, res, socket, state, server) {
   // If the user never called req.read(), and didn't pipe() or
   // .resume() or .on('data'), then we call req._dump() so that the
   // bytes will be pulled off the wire.
-  if (!req._consuming && !req._readableState.resumeScheduled)
```

Comment:
Should `_consuming` be removed?

Reply:
breaking?

Reply:
True! maybe add a TODO statement to remove it?

Reply:
Added TODO